sunpy/net/dataretriever/clients.py

```diff
@@ -9,6 +9,7 @@
 from .sources.norh import NoRHClient
 from .sources.rhessi import RHESSIClient
 from .sources.noaa import NOAAIndicesClient, NOAAPredictClient
+from .sources.kanzelhohe import KanzelhoheClient
```

Comment:
This is not a LightCurve, so to keep the comments meaningful I would put a blank line between the previous ones and add a comment like:
```python
# Import and register images sources
```